lib/src/blaze/sync_status.rs

```diff
@@ -13,7 +16,7 @@ pub struct BatchSyncStatus {
     pub trial_dec_done: u64,
     pub txn_scan_done: u64,
 
-    pub witnesses_updated: u64,
+    pub witnesses_updated: HashMap<ChannelNullifier, u64>,
```

Comment:
Let's update the name `ChannelNullfier` now that it's also used for `HashMap` keys.